### PR TITLE
Refactor: Style payment gateways settings page

### PR DESCRIPTION
**Pull request by pauloiankoski**
## Description

This pull request includes tooltips containing extra information for each tab on the payment gateways settings page. It also adds a notification dialog to inform about changes made since v3.

## Visuals

CleanShot 2023-09-11 at 17 07 12
CleanShot 2023-09-11 at 17 08 59
CleanShot 2023-09-11 at 17 09 18

## Pre-review Checklist

-   [ ] Acceptance criteria satisfied and marked in related issue
-   [x] Relevant `@unreleased` tags included in DocBlocks
-   [ ] Includes unit tests
-   [ ] Reviewed by the designer (if follows a design)
-   [x] [Self Review](https://give.gitbook.io/development-manual/devops/github/code-reviews#self-review) of code and UX completed

### CODE REVIEW

**Comment by pauloiankoski** [comment]
@jdghinson Could you review this work?

https://github.com/impress-org/givewp/assets/3921017/fdcbd2c6-1b4d-43f6-bb62-2846a736db2e

File: includes/admin/settings/class-settings-gateways.php [hunk 1]
```diff
@@ -479,6 +513,101 @@ public function render_enabled_gateways($field, $settings)
...
+                                    echo $gateway; ?>
+                                </div>
+                            <?php
+                            endforeach; ?>
+                        </div>
+                    <?php
+                    endif; ?>
+                    <button class="give-payment-gateway-settings-dialog__content-button"><?php
+                        esc_html_e('Got it', 'give'); ?></button>
+                    <a href="#" class="give-payment-gateway-settings-dialog__content-link"><?php
```
Comment (JoshuaHungDinh) [inline includes/admin/settings/class-settings-gateways.php:605]: Do we have a shortlink for this?

File: includes/admin/settings/class-settings-gateways.php [hunk 2]
```diff
@@ -479,6 +513,101 @@ public function render_enabled_gateways($field, $settings)
...
+                                         xmlns="http://www.w3.org/2000/svg">
+                                        <path fill-rule="evenodd" clip-rule="evenodd"
+                                              d="M7.063.986a1.531 1.531 0 0 1 1.872 0l.783.601.98-.129c.69-.09 1.354.294 1.62.935l.377.913.911.376h.002c.641.267 1.025.93.935 1.62l-.13.98.602.783a1.534 1.534 0 0 1 0 1.872l-.601.783.129.98c.09.69-.294 1.354-.935 1.62h-.002l-.91.377-.378.912a1.537 1.537 0 0 1-1.62.936l-.98-.13-.783.601a1.531 1.531 0 0 1-1.872 0l-.782-.6-.98.129a1.537 1.537 0 0 1-1.62-.936l-.377-.912-.911-.376H2.39a1.537 1.537 0 0 1-.935-1.621l.129-.98-.601-.783a1.533 1.533 0 0 1 0-1.872l.601-.782-.129-.98c-.09-.69.294-1.354.935-1.62l.002-.001.91-.376.377-.913a1.537 1.537 0 0 1 1.62-.935l.98.13.783-.602zm3.741 5.82a.667.667 0 0 0-.943-.943L7.333 8.392 6.47 7.53a.667.667 0 1 0-.943.943L6.86 9.806c.26.26.683.26.943 0l3-3z"
+                                              fill="#459948"></path>
+                                    </svg>
+                                    <?php
+                                    echo $gateway; ?>
```
Comment (JoshuaHungDinh) [inline includes/admin/settings/class-settings-gateways.php:596]: Is this supposed to include offline donations?

File: includes/admin/settings/class-settings-gateways.php (hunk 1 above)
Comment (JoshuaHungDinh) [reply includes/admin/settings/class-settings-gateways.php:605]: Also if its an external link: ```rel="noopener noreferrer``` https://lw.slack.com/archives/C02QJ8F07B3/p1668790561688639

File: includes/admin/settings/class-settings-gateways.php (hunk 2 above)
Comment (pauloiankoski) [reply includes/admin/settings/class-settings-gateways.php:596]: We are currently using the `getSupportedGateways` method as it's been used on another modal created a few weeks ago. We can create a new topic for that discussion but is not the purpose of this PR.

File: includes/admin/settings/class-settings-gateways.php (hunk 1 above)
Comment (pauloiankoski) [reply includes/admin/settings/class-settings-gateways.php]: Updated on f9634e927d486bd69e099338d4f50b5ef205b3d1

**Review by JoshuaHungDinh — APPROVED** [review]
Awesome, LGTM.

**Review by jdghinson — APPROVED** [review]
Looks good

**Pull request merged**